css-three-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  color: white;
+  font-family: sans-serif, 'Courier New', Courier;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+  justify-content: center;
+  align-items: center;
+}
+
+.row {
+  display: flex;
+  width: 90%;
+  justify-content: center;
+}
+
+.column-full {
+  display: flex;
+  justify-content: center;
```

Comment:
Column classes should only ever have `width` or `flex-basis` applied to them, these should be separate classes. Based on the fact that you made this `display: flex`, you likely want a `row` class element inside your `column-full` element.